components/RatingsTable/index.tsx

```diff
@@ -21,24 +21,60 @@ interface ColumnData {
 interface Props {
   data: Array<PlayerRatings | GoalieRatings>;
   columnData: Array<ColumnData>;
+  pagination?: boolean;
+  teamPage?: boolean;
   // isLoading: boolean;
 }
 
 function RatingsTable({
   data: players,
   columnData,
+  pagination = false,
+  teamPage = false,
 }: // isLoading
 Props): JSX.Element {
   // ! add loading state
   const data = useMemo(() => players, [players]);
 
   // ! add loading state
-  const columns = useMemo(() => columnData, []);
+  const columns = useMemo(() => {
+    // handle logic with teamPage
+    if(teamPage && columnData) {
+      // loop through columns to find the team column and remove it from the columnData
+      let i = columnData[0]["columns"].length
+      while(i--) {
+        if (columnData[0]["columns"][i].Header == "Team") {
+          delete columnData[0]["columns"][i]
+        }
+      }
+    }
+    return columnData
+  }, []);
+
+  const initialState = useMemo(() => {
```

Comment:
add pagination support.

Because the ratings table was never on the teams page, there was always pagination.  This logic removes the pagination on the team page.  Same logic as the stats table.